gulpfile.js

```diff
@@ -2,15 +2,38 @@ const gulp = require('gulp');
 const babelify = require('babelify');
 const browserify = require('browserify');
 const source = require('vinyl-source-stream');
+const gutil = require('gulp-util');
+const notify = require('gulp-notify');
 
-gulp.task('default', ['css', 'browserify']);
+const reportError = function (error) {
+  notify({
+    title: 'Task Failed [browserify]',
+    message: 'See console.',
+    sound: 'Sosumi' // See: https://github.com/mikaelbr/node-notifier#all-notification-options-with-their-defaults
+  }).write(error);
+
+  gutil.beep();
```

Comment:
Notification city! I get a beep sound, a `sosumi` sound, and *two* system notifications here! (My terminal shows a notification when you beep.) This is a bit much.

<img width="414" alt="multi-notifications" src="https://user-images.githubusercontent.com/74178/31651192-88684bd0-b2cf-11e7-944b-06c87c173919.png">

Can you please drop either the `notify()` or the `beep()` call?

Reply:
Strange, the beep() wasn't actually working for me at all, so I wasn't hearing anything from it. Removed because it was useless for me and annoying for you, so had to go with `notify()`. Also, realized how much I don't like `sosumi`, so changed it to `Purr`.